examples/gen_test_data/gen_test_data/generate_test_data_flow/flow.dag.yaml

```diff
@@ -0,0 +1,153 @@
+$schema: https://azuremlschemas.azureedge.net/promptflow/latest/Flow.schema.json
+environment:
+  python_requirements_txt: requirements.txt
+inputs:
+  text_chunk:
+    type: string
+    is_chat_input: false
+    default: Albert Einstein (14 March 1879 - 18 April 1955) was a German-born
+      theoretical physicist who is widely held to be one of the greatest and
+      most influential scientists of all time.
+  text_meta:
+    type: string
+    default: "{}"
+outputs:
+  question:
+    type: string
+    reference: ${validate_test_question.output}
+  suggested_answer:
+    type: string
+    reference: ${validate_suggested_answer.output.suggested_answer}
+  debug_info:
+    type: string
+    reference: ${generate_debug_info.output}
+nodes:
+- name: score_text_trunk_prompt
+  type: prompt
+  source:
+    type: code
+    path: score_text_trunk_prompt.jinja2
+  inputs:
+    context: ${inputs.text_chunk}
+  use_variants: false
+- name: validate_seed_question_prompt
+  type: prompt
+  source:
+    type: code
+    path: validate_question_prompt.jinja2
+  inputs:
+    question: ${validate_and_generate_seed_question.output.question}
+  use_variants: false
+- name: seed_question_prompt
+  type: prompt
+  source:
+    type: code
+    path: seed_question_prompt.jinja2
+  inputs:
+    context: ${inputs.text_chunk}
+  use_variants: false
+- name: generate_suggested_answer_prompt
+  type: prompt
+  source:
+    type: code
+    path: generate_suggested_answer_prompt.jinja2
+  inputs:
+    context: ${inputs.text_chunk}
+    question: ${validate_test_question.output}
+  use_variants: false
+- name: validate_and_generate_seed_question
```

Comment:
separate validate_text_trunk from the node.
And merge `validate_and_generate_seed_question` and `validate_and_generate_test_question` to one node.